mtp_cashbook/apps/cashbook/views.py

```diff
@@ -3,16 +3,38 @@
 from django.core.urlresolvers import reverse_lazy
 from django.utils.decorators import method_decorator
 from django.utils.translation import ugettext_lazy as _
-from django.views.generic import FormView
+from django.views.generic import FormView, TemplateView
+
+from mtp_cashbook.apps.mtp_auth.api_client import get_connection
 
 from .forms import ProcessTransactionBatchForm
 
 
+class DashboardView(TemplateView):
+    template_name = 'cashbook/dashboard.html'
+
+    @method_decorator(login_required)
+    def dispatch(self, request, *args, **kwargs):
+        self.client = get_connection(request)
+        return super(DashboardView, self).dispatch(request, *args, **kwargs)
+
+    def get_context_data(self, **kwargs):
+        context_data = super(DashboardView, self).get_context_data(**kwargs)
+
+        # new transactions == available + my pending
+        user = self.request.user
+        transaction_client = self.client.transactions(user.prison)
+        available = transaction_client.get(status='available')
+        my_pending = transaction_client(user.pk).get(status='pending')
+        context_data['new_transactions'] = available['count'] + my_pending['count']
+        return context_data
```

Comment:
As discussed in HipChat, I think it might be worth unit-testing this? Four cases I can think of: 0 available + 0 pending, some available + 0 pending, 0 available + some pending, some available + some pending.